### DISCUSSION TITLE
Bottomless Lava Bucket causes anti-cheat/liquid duplication debuffs

### DISCUSSION THREAD

**Discussion by R1cehat** [Pryaxis/TShock#1859] [Q&A]
Using normal buckets of lava is fine but when using the bottomless version causes even admins to get the debuffs.

**Comment by hakusaro** [comment]
This will be fixed in the next pre-release (fixed in https://github.com/Pryaxis/TShock/commit/7452d3c91d56e85e622b1e3d8ce2afdcbc913911).
